src/main/java/org/datavyu/plugins/nativeosx/NativeOSXMediaPlayer.java

```diff
@@ -0,0 +1,212 @@
+package org.datavyu.plugins.nativeosx;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.datavyu.plugins.MediaException;
+import org.datavyu.plugins.DatavyuMediaPlayer;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+import java.awt.*;
+import java.net.URI;
+
+abstract class NativeOSXMediaPlayer extends DatavyuMediaPlayer {
+
+  private static Logger logger = LogManager.getLogger(AVFoundationMediaPlayer.class);
+
+  protected static final int PRECISE_SEEK = 0;
```

Comment:
Add doc what the difference is between these seeks

Reply:
Done!